scripts/frame/main.js

```diff
@@ -91,10 +91,10 @@ frame.refreshPicture = function () {
 		let srcset = "";
 		let src = "";
 		this.frame.photo = null;
-		if (data.medium != "") {
+		if (data.medium !== "") {
 			src = data.medium;
 
-			if (data.medium2x && data.medium2 != "") {
+			if (data.medium2x && data.medium2 !== "") {
```

Comment:
I realize you "inherited" this typo but surely it should say `medium2x` in the second condition as well?

Reply:
yes.